src/biguint.rs

```diff
@@ -2845,6 +2865,138 @@ fn get_radix_base(radix: u32) -> (BigDigit, usize) {
     }
 }
 
+#[cfg(test)]
+#[cfg(has_i128)]
+quickcheck! {
```

Comment:
Let's move all of these to a standalone file, like `tests/quickcheck.rs`, which can gate on the optional feature en masse. We also won't need `has_i128` if we're only supporting quickcheck on newer versions to begin with.